src/os/ui/util/punyparent.js

```diff
@@ -97,27 +97,26 @@ os.ui.util.PunyParentCtrl.prototype.onResize_ = function() {
 os.ui.util.PunyParentCtrl.prototype.onThrottleResize_ = function() {
   if (this.element_) {
     var children = this.element_.children().toArray();
-    var childrenWidth = 0;
-    children.forEach(function(child) {
+    var childrenWidth = children.reduce((accumulator, child) => {
       var c = $(child);
       // ignore the resize trigger since thats the parent size
       if (!c.hasClass('resize-triggers')) {
```

Comment:
If this `if` case ever fails to pass, there is no `return accumulator` statement after, so the function is assumed to return `undefined`, which will result in `NaN` as the result rather than simply ignoring those elements. We should handle that case.

Reply:
Good catch! I moved the return outside of the if